.github/workflows/test.yml

```diff
@@ -1,4 +1,4 @@
-name: RDF differ test and lint
+name: test project
 
 on:
   push:
```

Comment:
we can specify on which branches this should be done, currently only on master, but we can have as many as we want (if needed)